ad-bank.md

```diff
@@ -0,0 +1,39 @@
+PostHog Ad Bank
+<hr/>
+
+## POSTHOG SELF-HOSTED
+
+* PostHog: product analytics software that’s fast, powerful, and easy to use - all completely self-hosted for maximum SOC 2 & HIPAA compliance. See it in action at posthog.com/product
+* Looking to self-host your own open-source product analytics platform? Look no further than PostHog - a fully-featured platform with no 3rd party cookies. Learn more at posthog.com
+* With PostHog’s self-hosted product analytics platform, your data stays on your infrastructure and you get full access to your production instance. See a demo today at posthog.com 
+* Self-hosted product analytics *and* complete privacy? We’ve got you covered. PostHog’s self-hosted app analytics platform is the perfect solution for privacy-conscious companies like yours. Deploy it free today at posthog.com/signup
+* PostHog’s open-source product analytics can be self-hosted and is great for small teams than want to understand usage in their apps. Get it free today at posthog.com/signup 
+* Looking for a free, open-source product analytics solution that you can self-host? Try PostHog today - and retain complete control over your data. Deploy it free at posthog.com/signup
```

Comment:
URL again. 
If you're wanting to point to the sign-up page, it's app.posthog.com/signup

Reply:
That sign-up page currently doesn't provide enough context for brand new users who've never heard of PosHog. I want to link to a page that gives them that info (currently /pricing) but without turning them off from the ad - you wanna learn more about a product before dropping stacks on it, so /signup is more intuitive/less pressure than /pricing.

Reply:
I'd suggest /product would be more useful than /pricing -- as you say, they want to learn more before paying. 

We could also consider a landing page for a developer demo, but maybe we're getting beyond the scope of general ad copy then

Reply:
Some of the ads link to /product, but others that reference "free" or pricing in some form should link to /signup (-> pricing) as there's a bit more urgency/intent behind them.